### PR TITLE
Fixing Bugs Last Run

### PR DESCRIPTION
**Pull request by lqueipom**
Modified:
- In verify() in ratingScript.js the user ID was saved differently than before, so we changed it accordingly. Now it is, userInfo.userID.
- I noticed a problem with the close button and the change of colors for the alerts in course.html so for now I am removing it until I am 100% it will not cause our code to break.
- Noticed a slight syntax problem in DataServletTest so I erased it. 
- Modified how we get translate boolean in DataServlet.
- Made some changes to isValidSubmission() in ratingScript.js that were made in another branch.

### CODE REVIEW

File: src/main/webapp/ratingScript.js [hunk 1]
```diff
@@ -103,13 +102,32 @@ async function getLatestRating() {
...
+    document.getElementById("rating-prof"),
+    document.getElementById("hours"),
+    document.getElementById("difficulty"),
+    document.getElementById("grade"),
+  ];
+
+  );
+  for (let i = 0; i < allFormFields.length; i++) {
+    // If any field is not field out properly.
+    if (!allFormFields[i].checkValidity()) {
```
Comment (ninaprabhu) [inline src/main/webapp/ratingScript.js:123]: for (let field of allFormFields)

**Comment by Tetheta** [comment]
Starting review

File: src/main/webapp/course.html [hunk 2]
```diff
@@ -269,18 +267,18 @@ <h1 class="p-4 text-center">Add Your Review</h1>
...
-            <span aria-hidden="true">&times;</span>
-          </button>
+        </div>
+
+        <div
+          id="submission-status-container"
+          class="alert alert-warning alert-dismissible fade show"
+          role="alert"
+        >
+          <strong id="submission-status-message"></strong>
```
Comment (Tetheta) [inline src/main/webapp/course.html:281]: FYI typically we bold with css, not with strong tags

File: src/main/webapp/ratingScript.js [hunk 3]
```diff
@@ -103,13 +102,32 @@ async function getLatestRating() {
...
+  ];
+
+  );
+  for (let field of allFormFields) {
+    // If any field is not field out properly.
+    if (!field.checkValidity()) {
+      return false;
+    }
+    // If no field was submitted the wrong way.
+    return true;
```
Comment (Tetheta) [inline src/main/webapp/ratingScript.js:127]: Shouldn't this be outside the for loop? Right now as soon as there's a valid field it's going to return true for the whole function.

Comment (lqueipom) [reply src/main/webapp/ratingScript.js:127]: Ohh that's weird! I think this got modified by accident in my previous PR changing that now! :)

File: src/main/webapp/course.html (hunk 2 above)
Comment (lqueipom) [reply src/main/webapp/course.html]: Good to know!! Just changed that as well!

**Pull request merged**